### PR TITLE
Loadouts/au army 2018 v8

### PR DESCRIPTION
**Pull request by AChesheireCat**
Added loadout: au_army_2018_v8

### CODE REVIEW

File: addons/tmf_loadouts/loadouts/au_army_2018_v8.hpp [hunk 1]
```diff
@@ -0,0 +1,572 @@
...
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+};
+
+class mtrtl : car {
+	displayName = "Mortar Team Leader";
+	secondaryWeapon[] = {"ace_csw_carryMortarBaseplate"};
+	items[] = { "MTR_GEAR" ,"ACE_RangeTable_82mm","ACE_MapTools"};
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/au_army_2018_v8.hpp:378]: No quotation marks around "MTR_GEAR"

File: addons/tmf_loadouts/loadouts/au_army_2018_v8.hpp [hunk 2]
```diff
@@ -0,0 +1,572 @@
...
+	uniform[] = {"FIR_Fighter_Pilot_Uniform_Flight"};
+	backpack[] = {};
+	linkedItems[] += {"ItemGPS"};
+	magazines[] = {
+		LIST_3("CUP_30Rnd_9x19_MP5"),
+		"SmokeShellPurple"
+	};
+};
+
+class eng : car {
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/au_army_2018_v8.hpp:534]: Add ACE mine detector

File: addons/tmf_loadouts/loadouts/au_army_2018_v8.hpp [hunk 3]
```diff
@@ -0,0 +1,572 @@
...
+	traits[] = {
+		"engineer",
+		"explosiveSpecialist"
+	};
+	backpack[] = {"B_Kitbag_rgr"};
+	magazines[] += {
+		LIST_4("ClaymoreDirectionalMine_Remote_Mag")
+	};
+	backpackItems[] = {
+		"MineDetector",
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/au_army_2018_v8.hpp:545]: Remove vanilla minedetector, add defusal kit, possibly entrenching tool and/or wirecutter. Also detonators

File: addons/tmf_loadouts/loadouts/au_army_2018_v8.hpp [hunk 4]
```diff
@@ -0,0 +1,572 @@
...
+		"ToolKit",
+		LIST_2("DemoCharge_Remote_Mag"),
+		LIST_2("SLAMDirectionalMine_Wire_Mag")
+	};
+};
+
+class engm : car {
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/au_army_2018_v8.hpp:552]: Same as above two

File: addons/tmf_loadouts/loadouts/au_army_2018_v8.hpp [hunk 5]
```diff
@@ -0,0 +1,572 @@
...
+		"ItemGPS"
+	};
+	backpackItems[] = {
+		LIST_3("CUP_100Rnd_TE4_LRT4_Red_Tracer_762x51_Belt_M")
+	};
+};
+
+class hmgg : car {
+	displayName = "HMG Gunner";
+	backPack[] = {"I_G_HMG_02_weapon_F"};
```
[the remark this answers is not in the document]
Comment (AChesheireCat) [reply addons/tmf_loadouts/loadouts/au_army_2018_v8.hpp:271]: Vanilla M2 has a gun shield which, while penetrated by any caliber above 7.62x39, is more aesthetically pleasing to me than the shield-less CUP. Just my opinion though, can change if necessary.

File: addons/tmf_loadouts/loadouts/au_army_2018_v8.hpp (hunk 1 above)
Comment (AChesheireCat) [reply addons/tmf_loadouts/loadouts/au_army_2018_v8.hpp:378]: Fix'd.

File: addons/tmf_loadouts/loadouts/au_army_2018_v8.hpp (hunk 3 above)
Comment (AChesheireCat) [reply addons/tmf_loadouts/loadouts/au_army_2018_v8.hpp:545]: Added all of the above and was able to keep the kit under 36kg.

File: addons/tmf_loadouts/loadouts/au_army_2018_v8.hpp (hunk 4 above)
Comment (AChesheireCat) [reply addons/tmf_loadouts/loadouts/au_army_2018_v8.hpp:552]: Ditto, same for this one too.

File: addons/tmf_loadouts/loadouts/au_army_2018_v8.hpp (hunk 2 above)
Comment (AChesheireCat) [reply addons/tmf_loadouts/loadouts/au_army_2018_v8.hpp:534]: The change request is showing me the Jet Pilot kit; are they supposed to have an ACE Mine Detector, ala VMM_3 or VMH_3?

File: addons/tmf_loadouts/loadouts/au_army_2018_v8.hpp (hunk 5 above)
[the remark this answers is not in the document]
Comment (Freddo3000) [reply addons/tmf_loadouts/loadouts/au_army_2018_v8.hpp:271]: Ah so its that one, I thought it was the scoped .50. Make sure that the new one doesn't have a DLC requirement.

File: addons/tmf_loadouts/loadouts/au_army_2018_v8.hpp (hunk 3 above)
Comment (Freddo3000) [reply addons/tmf_loadouts/loadouts/au_army_2018_v8.hpp:545]: 35 kg limit only really applies to regular squad roles. Specialists can go above.

File: addons/tmf_loadouts/loadouts/au_army_2018_v8.hpp (hunk 2 above)
Comment (Freddo3000) [reply addons/tmf_loadouts/loadouts/au_army_2018_v8.hpp:534]: Comment was posted on ENG/ENGM. Yeah, add either VMM or VMH.

File: addons/tmf_loadouts/CfgLoadouts.hpp [hunk 6]
```diff
@@ -54,4 +54,10 @@ class CfgLoadouts {
 		tooltip = "Author: Freddo\n\nNote: Low ammo count. Make sure resupply is available.";
 		#include "loadouts\tk_locals.hpp"
 	};
+	class au_army_2018 {
```
Comment (Freddo3000) [inline addons/tmf_loadouts/CfgLoadouts.hpp:57]: Move under `// Western Factions` a bit up

Comment (AChesheireCat) [reply addons/tmf_loadouts/CfgLoadouts.hpp]: fix'd

File: addons/tmf_loadouts/loadouts/au_army_2018_v8.hpp [hunk 7]
```diff
@@ -0,0 +1,581 @@
+// If you want to base a loadout on an existing one, this repository contains them all:
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/au_army_2018_v8.hpp:1]: Remove _v8 from file name

Comment (AChesheireCat) [reply addons/tmf_loadouts/loadouts/au_army_2018_v8.hpp]: I think I was able to change the name :D

File: addons/tmf_loadouts/CfgLoadouts.hpp [hunk 8]
```diff
@@ -53,6 +53,12 @@ class CfgLoadouts {
 		tooltip = "Author: Freddo";
 		#include "loadouts\ru_army_2010.hpp"
 	};
+	class ussr_vdv_1980 {
```
Comment (Freddo3000) [inline addons/tmf_loadouts/CfgLoadouts.hpp:56]: And now you accidentally added the VDV loadout to the Aussie branch. Get on TS and I'll show you how to manage branches in GitKraken.

**Pull request merged**